apis/DeviceInfo/DeviceInfo.json

```diff
@@ -138,7 +138,11 @@
       "type": "string",
       "enum": [
         "manufacturer1",
-	"RaspberryPi4"
+	      "RaspberryPi4",
```

Comment:
We don't want to add any proprietary information in this repo. Pls add them in the respective proprietary meta layers